lib/components/animation_component.dart

```diff
@@ -5,6 +5,7 @@ import 'package:flame/animation.dart';
 
 class AnimationComponent extends PositionComponent {
   Animation animation;
+  Paint paint;
```

Comment:
should we call it overridePaint to be consistent with Sprite?

Reply:
overridePaint is the name only on Sprite's render methods, the Sprite also has a property called paint. I think we should keep the paint name, as this is a property of the AnimationComponent.

I can change to overridePaint if you prefer, but in this case I think it will be a little confusing.

Reply:
Changed my mind, it is overridePaint now.